README.md

```diff
@@ -1,24 +1,101 @@
-# alertmanager-operator
+# Alertmanager Operator
 
 ## Description
 
-TODO: fill out the description
+The [Alertmanager](https://prometheus.io/docs/alerting/latest/alertmanager/)
+operator provides an alerting solution for the [Prometheus
+Operator](https://github.com/canonical/prometheus-operator). It is
+part of an Obervability stack in the [Juju](https://jaas.ai/) charm
+[ecosystem]((https://charmhub.io/). Alertmanager accepts alerts from
+Prometheus, then deduplicates, groups and routes them to the selected
+receiver, based on a set of alerting rules. These alerting rules may
+be set by any supported [charm](https://charmhub.io/) that uses the
+services of Prometheus by forming a relation with it.
```

Comment:
You've got a typo (Obervability) and bad markdown (double parens). I highly recommend you use the 'reference' style links when you're writing text that's link-heavy to make it easier to spot these errors.
For example,

```suggestion
The [Alertmanager] operator provides an alerting solution for the
[Prometheus Operator]. It is part of an Observability stack in the [Juju] charm
[ecosystem]. Alertmanager accepts alerts from Prometheus, then deduplicates, groups
and routes them to the selected receiver, based on a set of alerting rules. These
alerting rules may be set by any supported [charm] that uses the services of
Prometheus by forming a relation with it.

[Alertmanager]: https://prometheus.io/docs/alerting/latest/alertmanager/
[Prometheus Operator]: https://github.com/canonical/prometheus-operator
[Juju]: https://jaas.ai/
[ecosystem]: https://charmhub.io/
[charm]: https://charmhub.io/
```
although I'm not sure how much value the last `[charm]` link provides.

Something similar could be done below but I don't think they're this egregious.